<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOperateur</class>
 <widget class="QWidget" name="QOperateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpBox_opera">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Operator</string>
   </property>
   <widget class="QSpinBox" name="spnBox_AR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_DR1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_SL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_DR2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_RR">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_AR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_DR1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SL">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SL</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_DR2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_RR">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_velmod">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vel. mod</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_velmod">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_velocity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_velocity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_detune">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fine</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_coarse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coarse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Coarse</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_multiple">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_multiple">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>120</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mult.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_fine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-4</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lvldph">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lvl. Dph.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_lvldph">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_kbcurb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Curb</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rtdph">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rt. Dph.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_rtdph">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_adjTL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_adjTL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Adj. TL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBut_carrier">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>140</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Car.</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_carrmod">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Car./Mod.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_volume">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_kbcurb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-Lin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+Lin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-Exp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+Exp</string>
     </property>
    </item>
   </widget>
   <widget class="QEnveloppe" name="label_env">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>26</y>
      <width>118</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="hzSlider_volume">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEnveloppe</class>
   <extends>QLabel</extends>
   <header>qenveloppe.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
